tests/testthat/test-merge_sce_list.R

```diff
@@ -402,11 +465,29 @@ test_that("merging SCEs with same altExp features works as expected, with altexp
 
 
 
-test_that("merging SCEs with different altExp features works as expected, with altexps", {
+test_that("merging SCEs with 1 altexp but different features fails as expected, with altexps", {
 
-  # keep only the first 3 features from the first SCE
-  altExp(sce_list_with_altexp[[1]]) <- altExp(sce_list_with_altexp[[1]])[1:3]
+  # second list where 1 is missing all
```

Comment:
I liked the previous comment better!
```suggestion
  # keep only the first 3 features from the first SCE
```